src/properties/optionproperty.cpp

```diff
@@ -57,9 +57,7 @@ const std::vector<OptionProperty::Option>& OptionProperty::options() const {
 }
 
 void OptionProperty::addOption(int value, std::string desc) {
-    Option option;
-    option.value = value;
-    option.description = desc;
+    Option option = { std::move(value), std::move(desc) };
```

Comment:
We want the move though.. if we don't have a move, we would need a copy as the OptionProperty needs to store the values internally and can't rely on the reference being around forever